install/gcp/installer/forseti_server_installer.py

```diff
@@ -145,6 +148,28 @@ def create_firewall_rules(self):
             self.config.vpc_host_network,
             '0.0.0.0/0')
 
+        # Create firewall rule to open only port tcp:22 (ssh)
+        # to all the external traffics from the internet to ssh into client VM.
+        gcloud.create_firewall_rule(
+            self.format_firewall_rule_name(
+                'forseti-client-allow-ssh-external'),
+            [self.gcp_service_acct_email],
```

Comment:
This looks like the server's service account, not the client.

Also, should this be somewhere else?  What about the client installer?  How do you think?